addons/l10n_cl/views/report_invoice.xml

```diff
@@ -75,7 +75,6 @@
         </div>
     </template>
 
-
```

Comment:
Useless diff, even if you are right let's not change stuff for the pleasure of it ahah, the real reason why is that it break the github history and so it's more difficult to find the original pr :smile: